### ISSUE TITLE
Feature Request: Remove empty parameter list when collection initializer is used

### ISSUE THREAD

**Issue by Danghor** [JosefPihrt/Roslynator#603]
When using a non-empty collection initializer, the parameter list can be ommitted, if it is empty. I suggest showing a diagnostic here, to remove the parantheses.

**Product and Version Used**: Roslynator 2019 Visual Studio Plugin (2.1.3 from 08/08/2019)

**Steps to Reproduce**:
Use the following snippet:
```csharp
var foo = new List<string>() // RCSXXXX
{
    "foo"
};
```

**Actual Behavior**:
No diagnostic is produced.

**Expected Behavior**:
A fadeout diagnostic is produced for the parentheses and the codefix removes them:

```csharp
var foo = new List<string>
{
    "foo"
};
```

**Comment by JosefPihrt** [comment]
Fixed with 2c44fbc29a663efa58e6e80cd21dec03f06d571b
